kpops/components/base_components/kafka_connector.py

```diff
@@ -105,8 +105,7 @@ def helm(self) -> Helm:
     @property
     def _resetter_release_name(self) -> str:
         suffix = "-clean"
-        clean_up_release_name = self.full_name + suffix
-        return trim_release_name(clean_up_release_name, suffix)
+        return create_helm_release_name(self.full_name, suffix)
```

Comment:
this seems wrong, full_name is missing the suffix

Reply:
The `full_name + suffix` is not important here. Because I am taking the first 46 characters `name[:(52-len("-clean")]` of the input `name`. I can even add it to it if it is confusing.
I have added a new test in `test_kafka_connector`that checks that